mteb/tasks/BitextMining/multilingual/IWSLT2017BitextMinig.py

```diff
@@ -0,0 +1,115 @@
+from __future__ import annotations
+
+import datasets
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks import AbsTaskBitextMining, CrosslingualTask
+
+_LANGUAGES = {
+    "ar-en": ["ara-Arab", "eng-Latn"],
+    "de-en": ["deu-Latn", "eng-Latn"],
+    "en-ar": ["eng-Latn", "ara-Arab"],
+    "en-de": ["eng-Latn", "deu-Latn"],
+    "en-fr": ["eng-Latn", "fra-Latn"],
+    "en-it": ["eng-Latn", "ita-Latn"],
+    "en-ja": ["eng-Latn", "jpn-Jpan"],
+    "en-ko": ["eng-Latn", "kor-Hang"],
+    "en-nl": ["eng-Latn", "nld-Latn"],
+    "en-ro": ["eng-Latn", "ron-Latn"],
+    "en-zh": ["eng-Latn", "cmn-Hans"],
+    "fr-en": ["fra-Latn", "eng-Latn"],
+    "it-en": ["ita-Latn", "eng-Latn"],
+    "it-nl": ["ita-Latn", "nld-Latn"],
+    "it-ro": ["ita-Latn", "ron-Latn"],
+    "ja-en": ["jpn-Jpan", "eng-Latn"],
+    "ko-en": ["kor-Hang", "eng-Latn"],
+    "nl-en": ["nld-Latn", "eng-Latn"],
+    "nl-it": ["nld-Latn", "ita-Latn"],
+    "nl-ro": ["nld-Latn", "ron-Latn"],
+    "ro-en": ["ron-Latn", "eng-Latn"],
+    "ro-it": ["ron-Latn", "ita-Latn"],
+    "ro-nl": ["ron-Latn", "nld-Latn"],
+    "zh-en": ["cmn-Hans", "eng-Latn"],
+}
+
+_SPLITS = ["validation"]
```

Comment:
Why are we not using `test` split?

Reply:
@KranthiGV it was too big for direct use (8.5k samples). Happy to switch to the `test` split if that ends up being a better option.

Reply:
I think you can downsample like other datasets using `stratified_subsampling`

Reply:
Example: `InternationalCitizenshipQuestionsLegalBenchClassification` class

Reply:
Thanks @KranthiGV. That might be a good option for classification tasks (such as for instance `InternationalCitizenshipQuestionsLegalBenchClassification`) but the `stratified_subsampling` method needs a `label` parameters, and I am really not sure what `label` would we use in the context of a `BitextMining` task: https://github.com/embeddings-benchmark/mteb/blob/06a222b4aebbf43512658df4bcdd74d92a04a9fb/mteb/abstasks/AbsTask.py#L54